Example/XExtensionItem/ViewController.m

```diff
@@ -35,12 +33,10 @@ - (void)action {
                                                                                                        iOSAppURL:nil
                                                                                                    androidAppURL:nil];
         mutableParameters.alternateContentRepresentations = @{ @"text/html": @"<p><strong>Apple’s website markup</strong></p>" };
-        
-        [mutableParameters addEntriesToUserInfo:({
-            TumblrCustomShareParameters *tumblrParameters = [[TumblrCustomShareParameters alloc] init];
-            tumblrParameters.customURLSlug = @"want-this-for-xmas";
-            tumblrParameters;
-        })];
+        mutableParameters.tumblrParameters = [[XExtensionItemTumblrParameters alloc] initWithCustomURLPathComponent:@"want-this-for-xmas"
+                                                                                                  requestedPostType:XExtensionItemTumblrPostTypeLink
+                                                                                                        consumerKey:@"a9fal494mglspj33lffaj4920vks"
```

Comment:
Maybe make it more obvious that this is a fake value so people don’t try to use it and then open issues when it doesn’t work.